packages/css/src/components/heading/README.md

```diff
@@ -2,16 +2,16 @@
 
 # Heading
 
-Describes the content that follows it.
-Use Headings to allow the user to grasp the structure of the page quickly.
```

Comment:
Why did you remove this line? I like it

Reply:
I want to separate the component’s rationale (or goal) from its usage instructions. I restored it into the ‘guidelines’ section.